service/arm_templates/network_setup.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apiVersion": {
+      "type": "string",
+      "defaultValue": "2017-03-01",
+      "metadata": {
+        "description": "API version used by the Microsoft.Network/virtualNetworks resources."
+      }
+    },
+    "addressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/16",
```

Comment:
I set this as a string, but it can be changed to be an array if there's a need to support multiple CIDR blocks.

Reply:
I wonder how we can put comments in the parameters. Resources has `comments` property [1].

1: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates#comments-container

Reply:
I looked at the schema in that link but there doesn't seem to be a comments section for parameters. We could include comments in the metadata description field.

Reply:
FYI the CIDRs will need to be added to `azuretpr`. This is where we set them in `awstpr`. In the operator it will get the values from the cluster TPO when it runs the template.

https://github.com/giantswarm/awstpr/blob/master/spec/aws/vpc.go